po/de.po

```diff
@@ -390,9 +390,10 @@ msgstr[0] "%1 Stunde"
 msgstr[1] "%1 Stunden"
 
 #: ../qml/RemindersModel.qml:74
+#, fuzzy
 msgid "%1 minute"
 msgid_plural "%1 minutes"
-msgstr[0] "%1 Minute"
+msgstr[0] "ist die Anzahl der Minuten"
```

Comment:
I dont think this is correct.

Reply:
Corrected it on translate.ubports.com.

Reply:
You could either start the pr update manually or wait until Sunday for the new commit to show up.

Reply:
Thx! Im afraid he translated 1 string and then told Weblate to substitute it globally, I found it in at least 3 other repos in the same way... meh